src/rgw/rgw_rados.cc

```diff
@@ -3462,7 +3462,7 @@ int RGWRados::replace_region_with_zonegroup()
 
   int ret = rgw_get_system_obj(this, obj_ctx, pool ,oid, bl, NULL,  NULL);
   if (ret < 0 && ret !=  -ENOENT) {
-    ldout(cct, 0) << "failed to read converted: ret "<< ret << " " << cpp_strerror(-ret)
+    ldout(cct, 0) << __func__ << "failed to read converted: ret "<< ret << " " << cpp_strerror(-ret)
```

Comment:
need to add a space, or this will print `replace_region_with_zonegroupfailed to read converted:` (same for the other changes below)